x-pack/plugins/monitoring/public/legacy_shims.ts

```diff
@@ -143,6 +146,7 @@ export class Legacy {
         usageCollection,
       },
       appMountParameters,
+      infraLocators: infra?.locators,
```

Comment:
Minor thing but would it be more "comfortable" to write:
`infra.locators.logLocator`? 

Specially if at some point we want to add some other dependency from infra.

Reply:
Almost, I was thinking:
`
  infra: {
    locators: {
      logs:infra?.locators.logslocator
    }
  }
`